test/modules/standard/gmp/ferguson/gmp_dist_array.chpl

```diff
@@ -44,16 +41,27 @@ modulus.nextprime(modulus);
 writeln(modulus);
 sum.mod(sum, modulus);
 writeln(sum);
-delete modulus;
-delete sum;
+
 
 // Release the array elements.
+// TODO: remove this section once the records are being released correctly.
```

Comment:
For face-to-face.

Seems strange that this application needs to think about memory management.